internal/dump.go

```diff
@@ -2,16 +2,19 @@ package internal
 
 import (
 	"fmt"
-	"google.golang.org/grpc/metadata"
 	"time"
+
+	"google.golang.org/grpc/metadata"
 )
 
 type RPC struct {
-	Service  string      `json:"service"`
-	Method   string      `json:"method"`
-	Messages []*Message  `json:"messages"`
-	Status   *Status     `json:"error,omitempty"`
-	Metadata metadata.MD `json:"metadata"`
+	Service              string      `json:"service"`
+	Method               string      `json:"method"`
+	Messages             []*Message  `json:"messages"`
+	Status               *Status     `json:"error,omitempty"`
+	Metadata             metadata.MD `json:"metadata"`
+	MetadataRespHeaders  metadata.MD `json:"metadata_resp_headers"`
```

Comment:
```suggestion
	MetadataRespHeaders  metadata.MD `json:"response_headers"`
	MetadataRespTrailers  metadata.MD `json:"response_trailers"`
```

Reply:
I think it's better if we stick to the terminology (same as the request metadata is not `headers`). I don't have that much exp with gRPC so if I am wrong here let me know and I'll change it :)

Reply:
`metadata_response_headers` and `metadata_response_trailers` sounds good 👍

Reply:
Done 👍